dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -106,6 +106,17 @@ isoLevel.SERIALIZABLE.desc=Dirty reads, non-repeatable reads and phantom reads a
 isoLevel.SNAPSHOT.desc=Snapshot isolation for Microsoft SQL Server JDBC Driver and DataDirect Connect for JDBC driver.
 isoLevel.NONE.desc=Indicates that the JDBC driver does not support transactions.
 
+# Map Error
+mapError=Map error
+mapError.desc=Map a specific sqlCode or sqlState to an action such as a stale connection.
+
+mapError.sqlState=SQL State
+mapError.sqlState.desc=A string that follows either the XOPEN SQLstate conventions or the SQL:2003 conventions. Mutually exclusive with the sqlCode attribute.
```

Comment:
Based on tWAS documentation it seems like we only allow mapping a sqlCode or a sqlState, but not both. We could certainly allow both, but I'm not sure it would be necessary or if any users would need that level of granularity.

Reply:
userDefinedErrorMap only allows one or the other, but custom DataStoreHelper allows for taking both into account together.  By making the configuration in Liberty more flexible to cover both ways, you also ensure that we have the ability to let the user work around slight ordering differences that might exist between Liberty and how DataStoreHelper layers the identification of exceptions. Given that this can be trivially added, I don't really see why not to.